DQM/DTMonitorModule/python/dtTriggerEfficiencyTask_cfi.py

```diff
@@ -30,6 +25,5 @@
 # Modify for running in run 2 2016 data
 #
 from Configuration.Eras.Modifier_run2_common_cff import run2_common
-run2_common.toModify( dtTriggerEfficiencyMonitor, checkRPCtriggers = cms.untracked.bool(False),processDDU = cms.untracked.bool(False))
-
+run2_common.toModify( dtTriggerEfficiencyMonitor, inputTagTM = cms.untracked.InputTag('twinMuxStage2Digis:PhIn'))
```

Comment:
Should likely not be done for an L1T stage-1 workflow...

Reply:
Can you suggest an Era or equivalent modifier then? And a wf number to test it?
Thanks

Reply:
Hmm, perhaps `stage2L1Trigger` (so that it does not get applied to `stage1L1Trigger`)